### ISSUE TITLE
[LND] Add explanation on the circular rebalance modal

### ISSUE THREAD

**Issue by saubyk** [Ride-The-Lightning/RTL#407]
The circular rebalance modal can be confusing for a new user. We can add an information section on the modal to explain the process executing behind the modal for better conceptual understanding.
The information section should be similar to the one provided on the channel back up page and should have the below text.
```
Circular Rebalance is a payment you make to *yourselves*, to affect a relative change in the balances of two channels.
To do this, you send the payment out from the chosen channel and receive it back on the channel with the selected peer.
Please note, you will be paying routing fee to balance the channels
```
![Screen Shot 2020-08-09 at 6 00 02 PM](https://user-images.githubusercontent.com/39208279/89742806-2b3d1280-da6b-11ea-95fc-001715ff69ff.png)

**Comment by ShahanaFarooqui** [comment]
@saubyk We are already working on 5 step info-graphics to explain circular rebalancing. https://github.com/Ride-The-Lightning/RTL-Design/pull/46.

Won't it be an overkill to include the above message on the modal with infographics?

**Comment by saubyk** [comment]
@ShahanaFarooqui  please refer to my latest comment on the below PR:
https://github.com/Ride-The-Lightning/RTL-Design/pull/46#pullrequestreview-473054725

**Comment by ShahanaFarooqui** [comment]
Closed via PR #449.
